scripts/fetch_manifests.py

```diff
@@ -0,0 +1,94 @@
+"""
+This replaces `npe2 fetch --all`, which relied on PyPI scraping to retrieve all packages with the
+framework::napari classifier. This script instead gets information about active plugins from
+`classifiers.json`, which is generated from the PyPI BigQuery data (source of truth) via
+`bigquery.py`.
+
+Much of this code is taken from `npe2._inspection._fetch`, where `npe2 fetch --all` was implemented.
+"""
+import json
+import sys
+from concurrent.futures import ProcessPoolExecutor
+from pathlib import Path
+from typing import TypedDict
+
+from npe2 import fetch_manifest
+from packaging.version import Version
+
+PUBLIC = Path(__file__).parent.parent / "public"
+MANIFEST_DIR = PUBLIC / "manifest"
+CLASSIFIERS = PUBLIC / "classifiers.json"
+ERRORS = PUBLIC / "errors.json"
+
+
+def _latest_non_pre_release(versions: list[str]) -> str:
+    for v in versions:
+        if not Version(v).is_prerelease:
+            return v
+    return None
+
+
+def _current_manifest_is_valid(name: str, version: str) -> bool:
+    try:
+        existing_manifest = json.loads((MANIFEST_DIR / f"{name}.json").read_text())
+        return existing_manifest["package_metadata"]["version"] == version
+    except Exception:
+        return False
+
+
+class ManifestError(TypedDict):
+    name: str
+    version: str
+    error: str
+
+
+def _try_fetch_and_write_manifest(name: str, active_versions: list[str]) -> ManifestError | None:
+    version_to_fetch = _latest_non_pre_release(active_versions) or active_versions[0]
```

Comment:
This gets the manifest from the latest non-pre-release version, or just the latest version if no non-pre-release versions exist. Previous behavior may have been to just get the manifest for the latest version including pre-releases. I'm open to either.

Reply:
Hmm I guess my preference here would be for non-pre-release, as pre-release versions may still have bugs etc that get fixed in the release. @psobolewskiPhD thoughts?

Reply:
💯 with @DragaDoncila here. I dont think the plugin manager should pull pre-release. We could leave them for the hub, but even that is sketch.